From 5880a273a699f8aa6476e95f9fd9557ba7609b56 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 22 Jun 2018 13:38:12 +0800
Subject: [PATCH 4515/5242] MLK-19413-4 arm64: fsl-imx8q-arm2.dtsi: Enable
 pixel combiners

commit  fd20c190213b44126b6986b7fca4e627b3eaaf52 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables pixel combiners for the i.MX8qm ARM2 platform.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
index 1df4fdf..d0bedaa 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
@@ -957,6 +957,10 @@
 	status = "okay";
 };
 
+&pixel_combiner1 {
+	status = "okay";
+};
+
 &prg1 {
 	status = "okay";
 };
@@ -1021,6 +1025,10 @@
 	status = "okay";
 };
 
+&pixel_combiner2 {
+	status = "okay";
+};
+
 &prg10 {
 	status = "okay";
 };
-- 
1.7.9.5

